lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,141 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+        lines_of_code: integer?,
+        last_committed_at: string
+      )
+    end
+
+    def analyzer_version
+      Runners::VERSION
+    end
+
+    # This analyser use git metadata (.git/).
+    def use_git_metadata_dir?
+      true
+    end
+
+    def analyze(changes)
+      targets = changes.changed_paths.map { |x| relative_path(x)}
```

Comment:
[suggestion] Maybe `relative_path` is needless here:

```suggestion
      targets = changes.changed_paths
```

`Changes` does it already:

https://github.com/sider/runners/blob/0dc3eafffd38bba59d9fbb8394ebb5aa47eb4ff9/lib/runners/changes.rb#L80